project_task_parent_completion_blocking/tests/test_project_task_parent_completion_blocking.py

```diff
@@ -0,0 +1,66 @@
+from odoo.exceptions import ValidationError
+from odoo.tests.common import TransactionCase
+
+
+class TestParentChildBlock(TransactionCase):
```

Comment:
```suggestion
from odoo.addons.base.tests.common import BaseCommon


class TestParentChildBlock(BaseCommon):
```

With this change it is no longer necessary to use `with_context(mail_create_nolog=True)` (or similar).